build.gradle

```diff
@@ -1,6 +1,7 @@
 plugins {
     id 'java'
     id 'org.jetbrains.kotlin.jvm' version '1.3.40'
+    id 'org.jetbrains.kotlin.plugin.serialization' version '1.3.50'
```

Comment:
While I see the appeal of the compiler magic, I think this is more brittle and agree that it depends on implementation details a lot.